prover/Cargo.toml

```diff
@@ -24,6 +24,7 @@ log = { version = "0.4.14", default-features = false }
 anyhow = "1.0.75"
 num = "0.4.0"
 num-bigint = "0.4.3"
+rayon = "1.10"
```

Comment:
Don't use rayon directly, we can use `plonky2_maybe_rayon = { path = "../plonky2/maybe_rayon" }` .

Reply:
ok